### ISSUE TITLE
this module does not work if lua-nginx-module enabled

### ISSUE THREAD

**Issue by yangboyd** [fdintino/nginx-upload-module#113]
upload_pass location does not get accessed

**Comment by yangboyd** [comment]
it conflicts with that modules' ngx.req.read_body()

**Comment by fdintino** [comment]
Yes, I think this whole thing needs to be rewritten as a request body filter module. It's already not able to be used with other protocols (such as `uwsgi_pass` or `fastcgi_pass`)
